qemu/tests/cfg/in_place_upgrade.cfg

```diff
@@ -33,6 +33,8 @@
     check_rhel_ver = "cut -f1-2 -d. /etc/redhat-release | sed 's/[^0-9]//g'"
     RHEL.8:
         rmmod_module = "rmmod ip_set;rmmod nft_compat"
+    RHEL.9:
+        rmmod_module = "rmmod ip_set"
     s390,s390x:
         # no need to check in host, and we don't support RHEL.7.9 as the
         # host on s390x
```

Comment:
I would suggest combining them, because 8,9 and s390x all have ip_set, and combine them is easy to maintain.